client/starwhale/core/dataset/tabular.py

```diff
@@ -155,7 +156,8 @@ def __repr__(self) -> str:
 
     def asdict(self, ignore_keys: t.Optional[t.List[str]] = None) -> t.Dict:
         d = super().asdict(
-            ignore_keys=ignore_keys or ["annotations", "extra_kw", "data_type"]
+            ignore_keys=ignore_keys
+            or ["annotations", "extra_kw", "data_type", "data_uri"]
```

Comment:
why ignore `data_uri`?  data_uri is Link Type, which can already provide asdict function.

Reply:
On one hand expand `data_uri` as dict to datastore will cause optional None values , which will future cause pyarrow error.
On the other hand keep `data_uri` as an object is more intuitive than make it as an expanded dict.